pom.xml

```diff
@@ -3,7 +3,7 @@
     <parent>
     	<groupId>org.jenkins-ci.plugins</groupId>
     	<artifactId>plugin</artifactId>
-        <version>1.557</version>
+        <version>1.586</version>
```

Comment:
@KostyaSha would you expect this to be tested solely on 1.585.3 or on all versions inbetween ? Note that this patch only fixes an incompatibility with jenkins introduced in 1.586. There is no added functionality, so what would be the point supporting even older versions than current LTS ?